Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -1388,6 +1394,7 @@ public void AssignDefaultActions()
         /// <seealso cref="AssignDefaultActions"/>
         public void UnassignActions()
         {
+            defaultActions = default;
```

Comment:
If assigning null to default actions like this it seems like we will leak memory. DefaultInputActions seems to implement IDisposable so should be?
```

defaultActions?.Dispose();
defaultActions = default;
```

Reply:
Right - that's a good call, will change.